src/main/app/src/constants/Navbar.js

```diff
@@ -34,7 +34,7 @@ const Navbar = () => {
     <nav className="navbar is-spaced is-dark" role="navigation" aria-label="main navigation"> 
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="index.html"> WeTime <i className="fas fa-heartbeat"></i></a>
+          <a className="navbar-item" href="/"> WeTime <i className="fas fa-heartbeat"></i></a>
```

Comment:
We can use a react router link here so it doesn't reload the page. Can be done later.